source/release.rst

```diff
@@ -143,17 +136,26 @@ Or:
         git fetch origin master
         git checkout -b <branch-name> origin/master
 
-3. Run the command below. The ``conda/`` folder is where ``meta.yaml`` is located and the ``-c``
-tells conda the channels where the dependencies defined in ``meta.yaml`` can be found.
+3. Run ``conda env list``. There will be several paths listed.
+For each of those paths, run ``rm -rf <path>/miniconda3/conda-bld`` to ensure that you're building the latest package:
+
+4. Run the following commands:
 
     ::
 
-        conda build conda/ -c conda-forge
+        conda config --add channels conda-forge
+        conda config --set channel_priority strict
 
-4. When ``conda build`` is completed, you should see something like the example below.
-We only have one package of type ``noarch``, so it's compatible with both Python 2 and 3.
-But since we only officially support Python 3, it might not work with Python 2.
+5. Run ``conda build conda/ -c conda-forge``. The ``conda/`` folder is where ``meta.yaml`` is located and the ``-c``
+tells conda the channels where the dependencies defined in ``meta.yaml`` can be found. Keep the output of this command.
+We'll use it in step 7.
+
+6. Run ``conda search --info --use-local zstash``. The only dependency should be ``python >=3.6``. In particular,
+``python_abi`` should not be listed as a dependency.
 
+7. In the output of step 5, you should see something like the below.
+We only have one package of type ``noarch``, so it's compatible with both Python 2 and 3.
+Since we only officially support Python 3, it might not work with Python 2.
```

Comment:
```suggestion
7. In the output of step 5, you should see something like the below.
We only have one package of type ``noarch``, meaning it works on both Linux and OSX and is compatible with multiple versions of Python (3.6, 3.7, 3.8, etc.). 
Since we have constrained python versions to >= 3.6 in the dependencies, it will not work with Python 2 or any other version of Python <= 3.5.
```